processing/data_collection/gazette/spiders/ba_vitoria_da_conquista.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+import datetime as dt
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class BaVitoriaDaConquistaSpider(BaseGazetteSpider):
+    TERRITORY_ID = '2933307'
+    name = 'ba_vitoria_da_conquista'
+    allowed_domains = ['pmvc.ba.gov.br']
+    start_urls = ['http://dom.pmvc.ba.gov.br/']
+
+    def parse(self, response):
+        """
+        @url http://dom.pmvc.ba.gov.br/
+        @returns items 1
+        @scrapes date file_urls is_extra_edition territory_id power scraped_at
+        """
+        menu_xpath = '/html/body/div[2]/div[4]/div/div[1]'
```

Comment:
It is not a good practice to create xpaths with that granularity. For example, to get the years that are available in this page you can use:
```
available_years = response.css('div.visible-xs ul.dropdown-menu li a::text').re('(\d{4})')
```